### ISSUE TITLE
Resources consistently fail on all entries

### ISSUE THREAD

**Issue by lanmaster53** [WebBreacher/WhatsMyName#123]
Looks like some are issues with timing out and could be fixed by extending the timeout, but the others seem to be legit bugs.

```
...
[!] 'check_uri' (thread=Thread-20, object={'name': 'T-Mobile Support', 'check_url': 'https://support.t-mobile.com/people/{account}', 'account_existence_code': '200', 'account_existence_string': 'connections', 'account_missing_string': '404 ERROR', 'account_missing_code': '404', 'known_accounts': ['admin', 'support'], 'category': 'social', 'valid': True}).
...
[!] HTTPSConnectionPool(host='api.zooppa.com', port=443)  Max retries exceeded with url  /api/v3/users?filters%5B%5D=login+is+jason-domke-b2bb4918 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x10a7ffcc0>  Failed to establish a new connection  [Errno 8] nodename nor servname provided, or not known',)) (thread=Thread-17, object={'name': 'Zooppa', 'check_uri': 'https://api.zooppa.com/api/v3/users?filters[]=login+is+{account}', 'pretty_uri': 'http://community.zooppa.com/en-us/users/{account}', 'account_existence_code': '200', 'account_existence_string': 'login', 'account_missing_string': 'avatar', 'account_missing_code': '200', 'known_accounts': ['test', 'hacker'], 'category': 'social', 'valid': True}).
...
[!] HTTPConnectionPool(host='www.authorstream.com', port=80)  Read timed out. (read timeout=10) (thread=Thread-18, object={'name': 'authorSTREAM', 'check_uri': 'http://www.authorstream.com/{account}/', 'account_existence_code': '200', 'account_existence_string': 'Presentations on authorSTREAM', 'account_missing_string': '', 'account_missing_code': '404', 'known_accounts': ['test', 'john'], 'category': 'presos', 'valid': True}).
...
[!] HTTPSConnectionPool(host='www.mixcloud.com', port=443)  Read timed out. (read timeout=10) (thread=Thread-14, object={'name': 'Mixcloud', 'check_uri': 'https://www.mixcloud.com/{account}/', 'account_existence_code': '200', 'account_existence_string': 'is on Mixcloud', 'account_missing_string': 'Page Not Found', 'account_missing_code': '404', 'known_accounts': ['test', 'john'], 'category': 'music', 'valid': True}).
...
```

**Comment by WebBreacher** [comment]
Thanks for the submission. We will take a look at those and adjust the entries.

**Comment by WebBreacher** [comment]
fixed in #181
